core/common/src/main/java/alluxio/PropertyKey.java

```diff
@@ -2764,7 +2764,7 @@ public String toString() {
           .build();
   public static final PropertyKey USER_NETWORK_READER_CHUNK_SIZE_BYTES =
       new Builder(Name.USER_NETWORK_READER_CHUNK_SIZE_BYTES)
-          .setDefaultValue("64KB")
+          .setDefaultValue("1MB")
```

Comment:
What's the trade off for increasing the chunk size, would this negatively affect some workloads?

Reply:
In general more memory consumption per stream. I am running more concurrent IO tests to understand impact across different scenarios.

Reply:
This is just data path. For this particular property it only affects remote read.